### DISCUSSION TITLE
Will MLPerf intend to include the performance evaluation of LLMs

### DISCUSSION THREAD

**Discussion by tohneecao** [mlcommons/inference#1343] [General]
GPT3 and so on

**Comment by psyhtest** [comment]
Yes, this is planned for MLPerf Inference v3.1 in early August (and MLPerf Training v3.0 in late May).

**Reply by tohneecao** [reply]
Glad to hear this，may i ask which LLM and which Accelerator exactly.

**Comment by psyhtest** [comment]
It's (a clone of) GPT3 (175B parameters). It is up to MLCommons members whether to optimise it for their accelerator products.

**Comment by psyhtest** [comment]
Would you be interested to see results from NVIDIA? Qualcomm? Anyone else?

**Reply by tohneecao** [reply]
I prefer NVIDIA,  are there any results can show？

**Reply by psyhtest** [reply]
All official v3.1 results will be announced in late August - early September.
